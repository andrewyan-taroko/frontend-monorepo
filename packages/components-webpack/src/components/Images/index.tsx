import React, { ReactElement } from 'react';
import gameControllerJpg from '@/assets/images/game-controller.jpg';
import gameControllerPng from '@/assets/images/game-controller.png';
import spinnerGif from '@/assets/images/spinner.gif';
import spinnerSvg from '@/assets/images/spinner.svg';

const Images = (): ReactElement => (
  <div>
    <img src={gameControllerPng} alt="gameControllerPng" />
    <img src={gameControllerJpg} alt="gameControllerJpg" />
    <img src={spinnerGif} alt="spinnerGif" />
    <img src={spinnerSvg} alt="spinnerSvg" />
  </div>
);

export default Images;
