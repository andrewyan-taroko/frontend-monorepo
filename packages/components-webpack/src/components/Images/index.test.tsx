import React from 'react';
import renderer from 'react-test-renderer';
import Images from './index';

describe('Images', (): void => {
  it('should work', (): void => {
    const component = renderer.create(<Images />);
    expect(component.toJSON()).toMatchSnapshot();
  });
});
