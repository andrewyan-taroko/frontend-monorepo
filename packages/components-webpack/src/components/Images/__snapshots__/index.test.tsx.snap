// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Images initial props 1`] = `
<div>
  <img
    alt="gameControllerPng"
    src="game-controller.png"
  />
  <img
    alt="gameControllerJpg"
    src="game-controller.jpg"
  />
  <img
    alt="spinnerGif"
    src="spinner.gif"
  />
  <spinner.svg />
</div>
`;
